src/index.js

```diff
@@ -35,7 +59,20 @@ function logTwoResults() {}
  * @param {Object} person2 - A person's names. In the same format as person1.
  * @param {Object} person3 - A person's names. In the same format as person1.
  */
-function logThreeResultsCities() {}
+function logThreeResultsCities(...cities) {  //spread operator
+
+cities.forEach(({first,last})=>{    // iterates through the list
+  finder(first,last)          //  finds the user inputted first and last name
+  .then(([{city}])=>{       // isolates city and console.log's it
+    console.log(city)
+  }).catch((err)=>{            // throws an error if encountered
+      console.log(err);
+});
+
+});
+};
```

Comment:
This is a really clever solution! Nice job Milton. The only comment I have about it is that the **arguments** passed in to `logThreeResultsCities` are **_objects that represent people_**, **not cities**. That's why I changed the variable name on line 62 to "people"
```suggestion
function logThreeResultsCities(...people) {  //spread operator

people.forEach(({first,last})=>{    // iterates through the list
  finder(first,last)          //  finds the user inputted first and last name
  .then(([{city}])=>{       // isolates city and console.log's it
    console.log(city)
  }).catch((err)=>{            // throws an error if encountered
      console.log(err);
});

});
};
```